packages/mantine/src/hooks/useLoadingHandler.ts

```diff
@@ -0,0 +1,21 @@
+import {MouseEvent, MouseEventHandler, useState} from 'react';
+
+export const useLoadingHandler = (handler?: MouseEventHandler<HTMLButtonElement>) => {
```

Comment:
Worth unit testing?

Reply:
yeah I thought so. You confirmed. I will return this one to draft to avoid context switching with my current task and come back to this afterward to give it a bit of love and coverage.